### DISCUSSION TITLE
Customtkinter slider problem

### DISCUSSION THREAD

**Discussion by jeagh** [alejandroautalan/pygubu-designer#263] [Q&A]
I overcame the problem with the customtkinter checkbox and switch widgets but the problem with the slider seems to be due to a different reason. It occurs when the to and from values are set to anything other than left blank. If the values are left blank then the slider range is 0 - 1 and no error message is generated.

I have tried both int and double values but the problem persists.

In the code I currently have:
`
 self.slider_var: tk.IntVar = None
`
`
 self.builder.import_variables(self)
`

I did try tk.DoubleVar and set the to and from values to 1.0 and 100.0 but it didn't make any difference.

**Comment by alejandroautalan** [comment]
Hello @jeagh.

It was a bug on pygubu. Try installing pygubu v0.35.4
Let me know if you have more issues.

Regards
Alejandro A.

**Reply by jeagh** [reply]
Thank you Alejandro, I see that version 0.35.4 is not available through pip. Maybe I should wait until it is.

**Reply by alejandroautalan** [reply]
I uploaded it about 20 minutes ago. It will be available soon.
https://pypi.org/project/pygubu/0.35.4

**Reply by jeagh** [reply]
I now have pygubu version 0.35.4 but the problem persists.

Now, when the to and from values are entered into pygubu-designer the preview shows the slider at the 0, end as it should, instead of in the centre. However, the same error message is still generated.

**Reply by alejandroautalan** [reply]
Please post a minimal UI file example, so I can reproduce the error.

**Reply by jeagh** [reply]
Just to be sure that I post the correct file, do you want the xml.ui file or the generated .ui file?

**Reply by alejandroautalan** [reply]
The xml ui file.

In designer, try adding a variable  to the Slider.

**Comment by jeagh** [comment]
I should have ask, do you want the ui.py file or the xml file?

**Reply by alejandroautalan** [reply]
The xml ui file.

In designer, try adding a variable to the Slider.

**Comment by jeagh** [comment]
I do already have a variable. The xml file is 117 lines, is that OK to post here or would an email attachment be better?

**Reply by alejandroautalan** [reply]
Send it by email attachment.
